pkg/controllers/metrics/pod/controller.go

```diff
@@ -133,21 +165,65 @@ func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		},
 	})
 	c.recordPodStartupMetric(pod)
+	c.recordPodBoundMetric(pod, labels)
 	return reconcile.Result{}, nil
 }
 
 func (c *Controller) recordPodStartupMetric(pod *corev1.Pod) {
 	key := client.ObjectKeyFromObject(pod).String()
 	if pod.Status.Phase == phasePending {
+		podUnstartedTimeSeconds.With(map[string]string{
+			podName:      pod.Name,
+			podNameSpace: pod.Namespace,
+		}).Set(time.Since(pod.CreationTimestamp.Time).Seconds())
 		c.pendingPods.Insert(key)
 		return
 	}
 	cond, ok := lo.Find(pod.Status.Conditions, func(c corev1.PodCondition) bool {
 		return c.Type == corev1.PodReady
 	})
 	if c.pendingPods.Has(key) && ok {
-		podStartupDurationSeconds.Observe(cond.LastTransitionTime.Sub(pod.CreationTimestamp.Time).Seconds())
-		c.pendingPods.Delete(key)
+		if cond.Status != corev1.ConditionTrue {
+			podUnstartedTimeSeconds.With(map[string]string{
+				podName:      pod.Name,
+				podNameSpace: pod.Namespace,
+			}).Set(time.Since(pod.CreationTimestamp.Time).Seconds())
+		} else {
+			// Delete the unstarted metric since the pod is now started
+			podUnstartedTimeSeconds.Delete(map[string]string{
+				podName:      pod.Name,
+				podNameSpace: pod.Namespace,
+			})
+			podStartupDurationSeconds.Observe(cond.LastTransitionTime.Sub(pod.CreationTimestamp.Time).Seconds())
+			c.pendingPods.Delete(key)
+		}
+	}
+}
+func (c *Controller) recordPodBoundMetric(pod *corev1.Pod, labels prometheus.Labels) {
+	key := client.ObjectKeyFromObject(pod).String()
+	condScheduled, ok := lo.Find(pod.Status.Conditions, func(c corev1.PodCondition) bool {
+		return c.Type == corev1.PodScheduled
+	})
+	if pod.Status.Phase == phasePending {
+		// The podsScheduled condition may be True or Unknown while a pod is in the pending state. Only when the pod is not bound,
```

Comment:
I think we just want this metric to increase from the pod's creation timestamp. I don't think that we want to emit the difference between it being scheduled and it being bound. Ideally, we're just looking to see how long it took for Karpenter to take a pod from created to bound